docs/user-guide/nginx-configuration/configmap.md

```diff
@@ -160,6 +160,8 @@ The following table shows a configuration option's name, type, and the default v
 |[global-auth-response-headers](#global-auth-response-headers)|string|""|
 |[global-auth-request-redirect](#global-auth-request-redirect)|string|""|
 |[global-auth-snippet](#global-auth-snippet)|string|""|
+|[global-auth-cache-key](#global-auth-cache-key)|string|""|
+|[global-auth-cache-duration](#global-auth-cache-duration)|string|"10m"|
```

Comment:
why does this have a different default than https://github.com/kubernetes/ingress-nginx/pull/4278/files#diff-9bba411a7c28f1ef63c3a5339db109d5R385?

Reply:
that was wrong, thanks for catching that